css/navigation.css

```diff
@@ -0,0 +1,62 @@
+:root {
+    --secondary-background: #F7F7F7;
+}
+
+/* navbar styling */
+
+.navbar {
+    width: 90%;
+    padding: 20px 30px;
+}
+
+/* navbar links */
+
+.nav-links {
+    gap: 2.2rem;
+}
+
+/* navbar search */
+
+.searchbar {
+    display: flex;
+    flex: 0 1 350px;
+}
+
+.input-icon .material-icons {
+    top: 5px;
+    left: 10px;
+}
+
+.input-icon .input {
+    margin: 0;
+    border-radius: 4px;
```

Comment:
Try to use rems instead of px everywhere will help in responsiveness

Reply:
Actually, I didn't want them to change that is why I put pixel there. Otherwise, I have put rem where I want them to be responsive.